Chessington.GameEngine/Pieces/King.cs

```diff
@@ -10,7 +10,14 @@ public King(Player player)
 
         public override IEnumerable<Square> GetAvailableMoves(Board board)
         {
-            return Enumerable.Empty<Square>();
+            var minusOneToOne = new List<int>(new[] { -1, 0, 1 });
+
+            var moves = minusOneToOne.SelectMany(
+                x => minusOneToOne,
+                (northSouth, eastWest) => (northSouth, eastWest)
+            );
+
+            return ApplyMoves(moves, board);
```

Comment:
I'm not sure `ApplyMoves` is the most descriptive name, it doesn't really tell me what it does? I'd expect it to actually do the moves in the game, but that doesn't fit with the fact that it returns a list of squares, and that those squares are used as the return value for `GetAvailableMoves` 🙂

Reply:
Yep, renamed in 6c5cbf8 🙂